.env.example

```
@@ -16,13 +16,16 @@ TRANSMORPHER_DEV_MODE=false
 TRANSMORPHER_STORE_DERIVATIVES=true
 TRANSMORPHER_DISK_ORIGINALS=localOriginals
 TRANSMORPHER_DISK_IMAGE_DERIVATIVES=localImageDerivatives
+TRANSMORPHER_DISK_DOCUMENT_DERIVATIVES=localDocumentDerivatives
 TRANSMORPHER_DISK_VIDEO_DERIVATIVES=localVideoDerivatives
 #TRANSMORPHER_SIGNING_KEYPAIR=
 TRANSMORPHER_OPTIMIZER_TIMEOUT=10
 # More information: https://github.com/cybex-gmbh/transmorpher/tree/release/v0#configuration-options
 VIDEO_TRANSCODING_WORKERS_AMOUNT=1
 #TRANSMORPHER_VIDEO_ENCODER=cpu-hevc
 #TRANSMORPHER_VIDEO_ENCODER_BITRATE=9000k
+#TRANSMORPHER_DOCUMENT_REMOVE_METADATA=true
+#TRANSMORPHER_DOCUMENT_DEFAULT_PPI=100
```

May not be the best "default"/example. I would expect, that it is more likely to raise the default rasterization resolution, than lowering it any further. So i would just enter the configured default of 300 here, or raise it to like 600 instead.

```suggestion
#TRANSMORPHER_DOCUMENT_DEFAULT_PPI=600
```

---

set it to 600.